.github/workflows/deploy_dbt_docs.yaml

```diff
@@ -0,0 +1,71 @@
+name: deploy-dbt-docs
+
+on:
+  workflow_run:
```

Comment:
**suggestion (non-blocking):** I would add a [workflow_dispatch](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow) option on the off chance that we need to manually rebuild the pages from a run.

Reply:
Good call, done in ee4a7602b27bc7142e7f6bcc3e778187946dc3c6 👍🏻